docs/requirements.txt

```diff
@@ -1,3 +1,3 @@
 python-dp
 sphinx-rtd-theme
-sphinx<=3.1.2 # because of https://github.com/sphinx-doc/sphinx/issues/8074
\ No newline at end of file
+sphinx>=3.2.1
```

Comment:
Is this issue not relevant to us?

Reply:
https://github.com/OpenMined/PyDP/pull/238

Reply:
Nope: `broken sphinx 3.2.0 has been fixed in sphinx 3.2.1. `
We can safely upgrade.